packages/components/src/components/buttons/buttonStyleUtils.ts

```diff
@@ -33,6 +39,10 @@ export const getIconColor = (variant: ButtonVariant, isDisabled: boolean, theme:
             return theme.iconOnSecondary;
         case 'tertiary':
             return theme.iconOnTertiary;
+        case 'info':
+            return theme.iconAlertBlue;
+        case 'warning':
+            return theme.iconAlertYellow;
```

Comment:
optional: would be greate to turn switch to simple Record map

Reply:
Let's do it in another PR after merge/release